src/lib/components/events/event-carousel.svelte

```diff
@@ -127,6 +150,7 @@
 		-webkit-user-select: none;
 		-ms-user-select: none;
 		user-select: none;
+		visibility: hidden;
```

Comment:
Instead of `visibility: hidden;`, maybe using opacity might be able to help us achieve a good transition effect.

```css
opacity: 0;
```

Reply:
I replace the visibility with opacity, but the button is still clickable even when it's hidden. Using visibility: hidden fixes that problem so wouldn't it be better to do that?

Reply:
We can keep `visibility: hidden`. Adding the `opacity` will enable the transition effect.